### ISSUE TITLE
Error with RN 0.57: java.lang.String cannot be cast to AccessibilityDelegateUtil

### ISSUE THREAD

**Issue by farzeni** [callstack/react-native-paper#597]
### Current behaviour

With RN version 0.57, on android, in a new application created with _react-native init_ every component crash the application with the following error:

```
java.lang.String cannot be cast to com.facebook.react.uimanager.AccessibilityDelegateUtil$AccessibilityRole
setDelegate
    AccessibilityDelegateUtil.java:93
updateViewAccessibility
    BaseViewManager.java:260
onAfterUpdateTransaction
    BaseViewManager.java:266
updateProperties
    ViewManager.java:33
createView
    NativeViewHierarchyManager.java:269
execute
    UIViewOperationQueue.java:200
run
    UIViewOperationQueue.java:888
flushPendingBatches
    UIViewOperationQueue.java:1001
access$2400
    UIViewOperationQueue.java:46
doFrameGuarded
    UIViewOperationQueue.java:1061
doFrame
    GuardedFrameCallback.java:29
doFrame
    ReactChoreographer.java:134
doFrame
    ChoreographerCompat.java:105
run
    Choreographer.java:909
doCallbacks
    Choreographer.java:723
doFrame
    Choreographer.java:655
run
    Choreographer.java:897
handleCallback
    Handler.java:790
dispatchMessage
    Handler.java:99
loop
    Looper.java:164
main
    ActivityThread.java:6494
invoke
    Method.java
run
    RuntimeInit.java:438
main
    ZygoteInit.java:807

```
### Code sample

- create a new app with react-native init
- than install react-native-paper with 
```
$ yarn add react-native-paper
$ yarn add react-native-vector-icons
$ react-native link react-native-vector-icons
```
- add PaperProvider Component 
- add a Button or AppBar to the main component

### Your Environment

| software              | version
| --------------------- | -------
| android        | Oreo
| react-native          | 0.57
| react-native-paper    | 2.1.3
| node                  | 10.0
| npm or yarn           | 1.10.1

**Comment by satya164** [comment]
Maybe a bug in React Native? Related: https://github.com/facebook/react-native/issues/21754

**Comment by programreality** [comment]
Same issue

**Comment by RaulCorreia** [comment]
Same, this happens when I use the Button, as it says in the documentation.
<Button>Ok</Button>

**Comment by HasithaPriyasad** [comment]
same issue

**Comment by HasithaPriyasad** [comment]
While this is urgent i use some work around for this
<TouchableOpacity  onPress={()=>this.props.navigation.navigate('Signup')}>
        <Text style={styles.signupButton}> Sign up</Text>
</TouchableOpacity>

**Comment by Biplovkumar** [comment]
same here in 0.57.3

**Comment by satya164** [comment]
It's a bug in React Native 0.57.3. You need to downgrade to 0.57.1 or lower. I'm closing this issue since we can't fix it in paper.

**Comment by Biplovkumar** [comment]
@satya164  Sorry to say u give Proper solution then close any issue.

Solution :- 

1.delete your node_modules directory (rm -rf node_modules/)
2.npm i -S react-native@0.57.1  **or** yarn add  react-native@0.57.1
3.npm add @babel/runtime  **or** yarn  add @babel/runtime 
4.npm i schedule@0.4.0 --save-dev  **or** yarn  add schedule@0.4.0 --save-dev  
5.npm i  **or** yarn

**Comment by httol** [comment]
same issue,i think its a bug

**Comment by Trancever** [comment]
@httol It's bug in react-native. Check this comment - https://github.com/callstack/react-native-paper/issues/597#issuecomment-431038437
